src/v/datalake/record_multiplexer.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+#pragma once
+
+#include "container/fragmented_vector.h"
+#include "datalake/schemaless_translator.h"
+#include "json/writer.h"
+#include "model/record.h"
+
+// TODO Only needed for method definitions, not class. Move methods out of line
+// if possible
+#include "storage/parser_utils.h"
+
+#include <seastar/core/future.hh>
+
+#include <__concepts/same_as.h>
```

Comment:
use the header from the standard. im guess an editor auto added this?

Reply:
Fixed